rope/base/oi/doa.py

```diff
@@ -114,24 +118,55 @@ class _SocketReceiver(_MessageReceiver):
     def __init__(self):
         self.server_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self.data_port = 3037
+        self.key = uuid.uuid4().bytes
+
         while self.data_port < 4000:
             try:
-                self.server_socket.bind(('', self.data_port))
+                self.server_socket.bind(('127.0.0.1', self.data_port))
```

Comment:
can you use localhost or similar for better ipv6 something?

Reply:
localhost might work.